src/modules/blockchain/implementation/web3-service.js

```diff
@@ -145,7 +207,7 @@ class Web3Service {
                     from: this.config.publicKey,
                     to: contractInstance.options.address,
                     data: encodedABI,
-                    gasPrice: gasPrice || this.web3.utils.toWei('20', 'Gwei'),
+                    gasPrice: this.web3.utils.toWei('100', 'Gwei'),
```

Comment:
Wrap fetching gas price into separate command for now. It will be easier to implement dynamic gas price in the future